sdks/python/apache_beam/ml/inference/base.py

```diff
@@ -1477,26 +1638,31 @@ def __init__(
       model_handler: ModelHandler[ExampleT, PredictionT, Any],
       clock,
       metrics_namespace,
-      enable_side_input_loading: bool = False,
+      load_model_at_runtime: bool = False,
       model_tag: str = "RunInference"):
     """A DoFn implementation generic to frameworks.
 
       Args:
         model_handler: An implementation of ModelHandler.
         clock: A clock implementing time_ns. *Used for unit testing.*
         metrics_namespace: Namespace of the transform to collect metrics.
-        enable_side_input_loading: Bool to indicate if model updates
-            with side inputs.
+        enable_side_input_loading: Bool to indicate if model loading should be
```

Comment:
```suggestion
        load_model_at_runtime: Bool to indicate if model loading should be
```

Reply:
Did everything (including this) in a single commit to keep it clean